src/styles/pages/get-involved/donate.module.scss

```diff
@@ -0,0 +1,51 @@
+.sectionContainer {
+  display: flex;
+  padding-top: 32px;
```

Comment:
this gets overruled by the padding defined in line 6 so you can remove it if not needed, otherwise put it after the padding property to implement it. Just fyi, if the same property is defined multiple times in CSS, it implements the last definition.